runtime/Stdlib_Lazy.res

```diff
@@ -1,11 +1,13 @@
-// FIXME:
-//   This exists for compatibility reason.
-//   Move this into Pervasives or Core
-
 type t<'a> = lazy_t<'a>
```

Comment:
Not in scope of this PR, but once all "lazy" magic is removed from the compiler (see also #6960), this could just be an abstract type here (instead of a predefined type `lazy_t` from predef.ml).